cluster-autoscaler/cloudprovider/gce/gce_manager.go

```diff
@@ -113,14 +113,15 @@ type gceManagerImpl struct {
 	migInfoProvider MigInfoProvider
 	migLister       MigLister
 
-	location              string
-	projectId             string
-	templates             *GceTemplateBuilder
-	interrupt             chan struct{}
-	regional              bool
-	explicitlyConfigured  map[GceRef]bool
-	migAutoDiscoverySpecs []migAutoDiscoveryConfig
-	reserved              *GceReserved
+	location                        string
+	projectId                       string
+	templates                       *GceTemplateBuilder
+	interrupt                       chan struct{}
+	regional                        bool
+	explicitlyConfigured            map[GceRef]bool
+	migAutoDiscoverySpecs           []migAutoDiscoveryConfig
+	reserved                        *GceReserved
+	expanderEphemeralStorageSupport bool
```

Comment:
I don't think this is actually used anywhere, right?

Reply:
Correct, thanks for finding this!